.github/workflows/check.yaml

```diff
@@ -121,11 +120,15 @@ jobs:
         run: charmcraft -v pack
 
       - name: Run tests
-        run: ${{ matrix.test-command }}
+        run: |
+          export CHARM_PATH_JAMMY="$(pwd)/$(ls | grep '.*22.04.*\.charm$')"
```

Comment:
non-blocker:
As I said in another [PR](https://github.com/canonical/charm-local-users/pull/46#discussion_r1763295754) this approach at first glance seems that we have two different files, but we actually have just a single one. I think we could try to use the regex pattern to try to find a single file for both `22.04` and `20.04` and have a single variable

Reply:
Yes so I'm trying to find a way that upholds:

- is consistent across all our func test workflows
- will work for charms with a single file output
- will scale easily to charms with multiple file outputs

So far this seems a suitable method.  I could put a comment clarifying in this place, but if it's ok, I'll put the comment in https://github.com/canonical/solutions-engineering-automation/pull/84 (where we add it to automation), otherwise it'll be duplicating effort across all the PRs.

Reply:
Also note that as soon as we split the bases, there will be two files output, then later there will be three when we add noble support.  So I'd rather this now, than finding a method that works for a single file and having to immediately change it back when we split the bases.

Reply:
I am ok with the change. But please add some comment to explain why this is required on the check.yaml file for future follow up.